src/views/Rules/ListRules/index.jsx

```diff
@@ -416,6 +417,35 @@ function ListRules(props) {
           // diff viewer
           height += diffedProperties.length * 21 + theme.spacing(5);
         }
+
+        if (Object.keys(rule.scheduledChange.required_signoffs).length > 0) {
+          const requiredRoles = Object.keys(
+            rule.scheduledChange.required_signoffs
+          ).length;
+          const nSignoffs = Object.keys(rule.scheduledChange.signoffs).length;
+
+          // Add space for the padding above the summary,
+          // and at the bottom of the section
```

Comment:
Which summary. The comment?

Reply:
Summary meaning SignoffSummary. This is to account for the padding that we're passing to it via className.

Reply:
Ah okay. I understand now. Makes sense.